homeassistant/components/dlna_dmr/__init__.py

```diff
@@ -3,39 +3,13 @@
 
 from homeassistant import config_entries
 from homeassistant.components.media_player.const import DOMAIN as MEDIA_PLAYER_DOMAIN
-from homeassistant.const import CONF_PLATFORM, CONF_URL
 from homeassistant.core import HomeAssistant
-from homeassistant.helpers.typing import ConfigType
 
-from .const import DOMAIN, LOGGER
+from .const import LOGGER
 
 PLATFORMS = [MEDIA_PLAYER_DOMAIN]
 
 
-async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
-    """Set up DLNA component."""
-    if MEDIA_PLAYER_DOMAIN not in config:
-        return True
-
-    for entry_config in config[MEDIA_PLAYER_DOMAIN]:
-        if entry_config.get(CONF_PLATFORM) != DOMAIN:
-            continue
-        LOGGER.warning(
```

Comment:
What is normally the deprecation time for configuration changes, or rather showing warnings about it? If this is merged its only one month, since #55267, that the `dlna_dmr` component was upgraded.

Reply:
We're moving the warning to the media_player platform.

Reply:
The depreciation time is normally two releases (months).